modules/neurons/training/mntp_trainer.py

```
@@ -495,6 +512,18 @@ def _write_summary(self, summary: dict[str, Any]) -> None:
         summary_path = self.output_dir / "training_summary.json"
         summary_path.write_text(json.dumps(summary, indent=2, sort_keys=True))
 
+    def _compute_file_checksum(self, path: Path) -> str:
+        try:
+            data = path.read_bytes()
+        except OSError as exc:
+            logger.error(
+                "Failed to compute checksum for artifact",
+                extra={"path": str(path)},
+                exc_info=exc,
+            )
```

Ruff (0.13.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)